AppHandling/Run-AlPipeline.ps1

```diff
@@ -1655,6 +1659,8 @@ Measure-Command {
     $Parameters = @{
         "missingDependencies" = @($unknownTestAppDependencies | Where-Object { $missingTestAppDependencies -contains "$_".Split(':')[0] })
         "appSymbolsFolder" = $appSymbolsFolder
+        "installedApps" = $installedApps
+        "installedCountry" = $artifactUrl.Substring($artifactUrl.LastIndexOf('/')+1)

```

Comment:
I see some duplication of code in '$artifactUrl.Substring($artifactUrl.LastIndexOf('/')+1)'. Move out to separate function?

Reply:
You will find a lot of things like this in Run-AlPipeline. In the end, the function is being deprecated, I don't want to start cleaning up things like this now.